aspnetcore/signalr/javascript-client.md

```diff
@@ -262,46 +262,6 @@ The following code demonstrates a typical manual reconnection approach:
 
 A real-world implementation would use an exponential back-off or retry a specified number of times before giving up.
 
-## Troubleshoot WebSocket handshake errors
```

Comment:
Is this going to break incoming links? If so, we should leave it here if `< aspnetcore-5.0'.

Reply:
This was added a couple days ago, so no